extensions/amp-analytics/amp-analytics.md

```diff
@@ -839,6 +839,8 @@ not specified, it waits for element's [`ini-load`](#ini-load) signal. See
 for that signal before sending the event. This is useful, for example, in
 sending analytics events when the page is closed.
 
+Tracking non-AMP elements are best effort. For example, tracking a `<div>` element that contains an `<amp-iframe>`, may not accurately wait for the iframe to load before sending the signal out.
```

Comment:
Move the paragraph to where we document `waitFor`. 

Also I wouldn't say it's best effort, because we always track them.